plugins/connectrpc/swift-mocks/v1.0.0/buf.plugin.yaml

```diff
@@ -0,0 +1,29 @@
+version: v1
+name: buf.build/connectrpc/swift-mocks
+plugin_version: v1.0.0
+source_url: https://github.com/connectrpc/connect-swift
+integration_guide_url: https://connectrpc.com/docs/swift/testing
+description: Generates mocks that are compatible with Connect-Swift clients.
+deps:
+  - plugin: buf.build/apple/swift:v1.28.2
+  - plugin: buf.build/connectrpc/swift:v1.0.0
+output_languages:
+  - swift
+registry:
+  swift:
+    deps:
+      - source: https://github.com/connectrpc/connect-swift.git
+        package: connect-swift
+        products: [ ConnectMocks ]
+        swift_versions: [ ".v5" ]
+        version: 1.0.0
+        platforms:
+          macos: v10_15
+          ios: v12
+          tvos: v13
+          watchos: v6
```

Comment:
Spoke to @rebello95 to confirm these are still right - we were missing watchos platform: https://github.com/connectrpc/connect-swift/blob/a8c984a1077f78e94e0884c5c11683a7f684f92c/Package.swift#L21-L26